packages/bot-skeleton/src/utils/local-storage.js

```diff
@@ -44,24 +44,24 @@ export const saveWorkspaceToRecent = (xml, save_type = save_types.UNSAVED) => {
         workspaces.pop();
     }
 
-    localStorage.setItem('saved_workspaces', LZString.compress(JSON.stringify(workspaces)));
+    localForage.setItem('saved_workspaces', LZString.compress(JSON.stringify(workspaces)));
 };
 
-export const getSavedWorkspaces = () => {
+export const getSavedWorkspaces = async () => {
     try {
-        return JSON.parse(LZString.decompress(localStorage.getItem('saved_workspaces'))) || [];
-    } catch (e) {
+        return JSON.parse(LZString.decompress(await localForage.getItem('saved_workspaces'))) || [];
+    } catch (err) {
```

Comment:
What's the difference between `e` and `err`?

Reply:
that was a typo, sorry